org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/Bounds.java

```diff
@@ -0,0 +1,99 @@
+package edu.mit.csail.sdg.ast;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import edu.mit.csail.sdg.alloy4.ConstList;
+import edu.mit.csail.sdg.alloy4.Err;
+import edu.mit.csail.sdg.alloy4.ErrorWarning;
+import edu.mit.csail.sdg.alloy4.Pos;
+
+public class Bounds extends Expr {
+
+    /** The label for this sig; this name does not need to be unique. */
+    public final String label;
+
+    public final ArrayList<CommandScope> scope;
+    
+    public final ArrayList<ExprVar> funcs;
+    
+    public final Expr fact;
+    
+    public Bounds(String label){
+        super(Pos.UNKNOWN, null);
+        this.label = label;
+        scope = new ArrayList<CommandScope>();
+        funcs = new ArrayList<ExprVar>();
+        fact = null;
+
+        
+    }
+
+    public Bounds(Pos pos, String label, List<CommandScope> list){
+        super( Pos.UNKNOWN,Type.FORMULA);
```

Comment:
let's try to keep spacing consistent.  For example, here you used `super( Pos...)` and above you used `super(Pos...)`

Reply:
fixed